hw/xbox/mcpx_apu.c

```diff
@@ -1612,6 +1652,21 @@ static void mcpx_apu_register(void)
 }
 type_init(mcpx_apu_register);
 
+#if USE_APU_THREAD
+static void *mcpx_apu_frame_thread(void *arg)
+{
+    volatile MCPXAPUState *d = MCPX_APU_DEVICE(arg);
+
+    while (!d->exiting) {
+        int should_run = (d->regs[NV_PAPU_SECTL] & NV_PAPU_SECTL_XCNTMODE) >> 3;
```

Comment:
nit: This is an unsuitable variable name

Reply:
Agree